Shared/Models/Message Model/Messageable.swift

```diff
@@ -32,7 +33,7 @@ protocol Messageable {
     var color: Color { get }
     var kind: MessageKind { get }
     var isDeleted: Bool { get }
-    var conversationId: String { get }
+    var recentReplies: [Messageable] { get }
```

Comment:
Adding this to messageable so we can determine if a cell is the most recently sent one.